src/components/search-dialog.tsx

```
@@ -0,0 +1,307 @@
+        // Then fetch media results
+        const response = await fetch(`/api/search/multi?query=${encodeURIComponent(debouncedQuery)}`)
+        if (!response.ok) throw new Error('Search failed')
+
+        const data = await response.json()
+        console.log(`Got search results for "${debouncedQuery}":`, data)
+        const mediaResults = data.results.map((item: any) => ({
+          title: item.title || item.name || '',
+          media_type: mapMediaType(item.media_type),
+          path: `/${item.media_type}/${item.id}`,
+          excerpt: item.overview
+            ? item.overview.substring(0, 100) + '...'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)